apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -74,13 +74,15 @@
 // (this allows it to take precedence over the component library button styles)
 button.button-kebab {
   display: none;
-  height: 20px;
-  width: 20px;
-  margin: 2px 2px 2px 0;
 }
 
-div.row:hover .button-kebab {
+div.draggable:hover .button-kebab,
+div.draggable:focus-within .button-kebab {
```

Comment:
Added this to draggable instead of row because draggable is clickable/focusable, and row is not